Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/execution/Context.java

```diff
@@ -566,7 +566,11 @@ private FunctionDef resolveFunctionDesc(FunctionDesc functionDesc, List<Object>
         }
 
         for (var i = 0; i < arguments.size(); i++) {
-            isMatch = isType(resolveType(arguments.get(i)), operands.get(i));
+            if (arguments.get(i) instanceof NamedTypeSpecifier) {
```

Comment:
suggestion: There isn't enough information available to unambiguously resolve the operator, so the behavior should be to fail if overloads are detected and there's no type information.